UnitsNet/UnitDefinitions/Altitude.json

```diff
@@ -0,0 +1,24 @@
+﻿{
```

Comment:
As mentioned previously in the pull request I believe this is the same physical quantity as `Length` or am I missing something. I have to admit I get confused by all the usages in aviation of masl.

Reply:
Altitude is not redundant to length because, even if from a dimensional point of view there is no difference, the altitude implicitly contains the information about the reference system.
Altitude is a widespread absolute measure of the height of a point.
In fact, in order to determine the altitude of a point, you need to have the reference datum of the level of sea (please see also: https://en.wikipedia.org/wiki/Altitude). 
If you use only length, you are going to miss the information about the reference, hence it won't be possible to detect an absolute height.

Moreover, the context is important. 
Who uses this measure is used to think with a reference in mind (the sea level) and then expects to work with this kind of measure unit.

Reply:
@angularsen Some input from you would be good on this

Reply:
In my opinion this would be out of scope of UnitsNet. Wouldn't it would make more sense to add another property with the reference datum level and save these 2 `Length` inside a new struct or class in you project?

Reply:
We chose to use this library because it provides everything we need out of the box. We don't have to create any kind of wrapper to adapt, thus that solution of yours doesn't really fit our needs.

The concept should be pretty similar to TemperatureDelta. We're always talking about temperature, but the delta means you're setting a reference system and starting your measurements from a specific value.

More, I must admit _Altitude_, in the terms we presented it to you, is very highly used in renewable technology context, much more than the other physical quantities we added, actually.

Reply:
I have only skimmed the wiki and are not very familiar with the domain of altitude and units.

1. Are there other reference datums widely used, something like MASL 1970 or MASL 1990? I have not researched this, but the argument for not using Length only holds water if there are actually multiple reference datums to consider that would be represented as multiple altitude units. If they exist and are widely used, then they should be part of this PR.
2. Are there operator overloads for other quantities that would be useful? Example: `Length / Duration = Speed`

We generally want to avoid single-unit quantities in the library, because I think the value of these are limited and could just as easily be defined per-application. I personally think UnitsNet shines when you have multiple units to convert between, operator overloads that construct other quantities.

Reply:
I understand your point.
I searched in order to find alternative units definitions to convert to/from masl and wikipedia said: 
_"Feet above sea level is the most common analogue for metres above sea level in the American measurement system."_
Said that, we never had occurrences of that unit but it could be added as well. 

If you keep struggling with the idea, what do you think if we add _masl_ among the abbreviations of Length - Meter? It's a bit odd, but could it work?

Reply:
I definitely do not want to add it to Length, that would be very strange indeed.

We could add the feet variant, but again, this feels like Length to me - just that this particular length/distance has a semantic meaning (distance to mean sea level). I feel this definition belongs in an application rather than in a shared library of units. It doesn't seem to benefit users much except having a separate type for it to help encode the meaning, and you can achieve that yourself with a wrapper type - or use C# type aliasing (`using Altitude = UnitsNet.Length;`) although that doesn't help you distinguish it from Length types. 

Seeing that there are no other scales or "datums" used, I argue that altitude should be kept out of the library to avoid adding too many of these single-unit quantities.
I propose you remove Altitude from this PR so we can merge in the rest of it, and if you still feel strongly about Altitude or have new arguments for adding it we can continue that conversation in another issue. However, I think we have covered most arguments for and against by now.

Reply:
Ok, we'll proceed as requested.